### ISSUE TITLE
[Docker Hub] Hub activity dashboard

### ISSUE THREAD

**Issue by pkennedyr** [docker/roadmap#19]
**Tell us about your request**
Dashboard with an overview of all recent activity within a Docker Hub account.

**Which service(s) is this request for?**
Docker Hub

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
Hub customers has cited a desire to have an overview of a repository's activity, including a list of recent tag updates, last runs of automated builds, image vulnerabilities, and users who have recently pushed to the repo.

**Are you currently working around this issue?**
N/A

**Additional context**
N/A

**Attachments**
N/A

**Comment by tehautanop** [comment]
Should this activity data in the dashboard also include the users last login info - which would be helpful for license management purposes ?

**Comment by tehautanop** [comment]
To manage the licenses as Docker Business license owner, SSO and SCIM enabled, we really need the ability to see which users have logged into Docker Desktop (or hub) and their last activity.

**Comment by fschroder** [comment]
To add to this request other activity would be useful as well, for example: pulls per repository / image, storage used for the whole account, etc.

**Comment by martinalbert** [comment]
Hey @Bkblodget , @ajthilakan , @sheltongraves, bit on the similar matter, I wonder what's the status on this from API standpoint, I can see `last_logged_in_at` field on organization member object, but it seems to be set to null at all times. Audit logs are available through the UI as well as API, but it does not contain any login activities unfortunately.

**Comment by etugarinova** [comment]
Hi @martinalbert! We would like to work on the API to expose things like `last_logged_in_at` and # of active users amongst other metrics in Q1, but it is waiting on a priority review. Until that is resolved, this work is parked.

**Comment by martinalbert** [comment]
Got it, thank you @etugarinova! Is there maybe a chance you could give us a rough ETA?

**Comment by etugarinova** [comment]
Rough eta would be around May/June but that's very speculative
